db/migrate/20221124073632_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUser < ActiveRecord::Migration[7.0]
```

Comment:
There is a migration error so I was not able to migrate the database. Kindly make sure that there is no database errors.

![mig](https://user-images.githubusercontent.com/35504149/203855238-45cf1440-97da-4ae1-8bb8-38b00037d131.png)